splitio/engine/impressions.py

```diff
@@ -1,160 +1,18 @@
 """Split evaluator module."""
-import threading
-from collections import defaultdict, namedtuple
 from enum import Enum
 
-from splitio.models.impressions import Impression
-from splitio.engine.hashfns import murmur_128
-from splitio.engine.cache.lru import SimpleLruCache
 from splitio.client.listener import ImpressionListenerException
-from splitio import util
-
-
-_TIME_INTERVAL_MS = 3600 * 1000  # one hour
-_IMPRESSION_OBSERVER_CACHE_SIZE = 500000
-
 
 class ImpressionsMode(Enum):
     """Impressions tracking mode."""
 
     OPTIMIZED = "OPTIMIZED"
     DEBUG = "DEBUG"
 
-
-def truncate_time(timestamp_ms):
```

Comment:
why are we removing all these classes / functions from this module? I would think that they belong to the impressions domain?